region/maridia/outer/Crab Gate Room.json

```
@@ -979,6 +1015,42 @@
       },
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 2],
+      "name": "R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Gravity",
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 100}}
+              ]},
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 3}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 40}}
+              ]}
+            ]}
+          ]}
+        ]},
+        {"canShineCharge": {"usedTiles": 42, "steepDownTiles": 1, "openEnd": 1}},
```

I would count this as max. Mostly if youre counting over 35 tiles, it would be for a low-level or cross-room strat. Technically it'd be 43 tiles with a steep up tile (which is longer) and 0 open ends
```suggestion
        "h_shinechargeMaxRunway",
```